server/services/transcation-service.js

```diff
@@ -0,0 +1,114 @@
+const Transaction = require('../models/transaction-model');
```

Comment:
This is transcation-service, (spelled wrong, i think we can delete this)

Reply:
Deleted it